app/services/analytics.rb

```diff
@@ -53,6 +53,7 @@ def uuid
   PASSWORD_RESET_TOKEN = 'Password Reset: Token Submitted'.freeze
   PHONE_CHANGE_REQUESTED = 'Phone Number Change: requested'.freeze
   PHONE_CHANGE_SUCCESSFUL = 'Phone Number Change: successful'.freeze
+  PROFILE_ENCRYPTION_INVALID = 'Profile encryption invalid'.freeze
```

Comment:
Can we make the capitalization more consistent? What about "Profile Encryption: invalid" (or Decryption based on other parts of the PR)